lib/js/media.js

```diff
@@ -637,7 +637,9 @@ window.tsfMedia = function () {
 		if ( ! imageId ) return;
 
 		const preview      = document.getElementById( `${imageId}-preview` ),
-			  imageWarning = document.getElementById( `${imageId}-image-warning` );
+			  imageWarning = document.getElementById( `${imageId}-image-warning` ),
+			  selectButton = document.getElementById( `${imageId}-select` ),
```

Comment:
This variable is used only once. So, don't create it, but chain below directly into inputType (now: imageType).